pkg/ncmpcpp

```diff
@@ -0,0 +1,38 @@
+[mirrors]
+http://github.com/arybczak/ncmpcpp/archive/b7da5d1b9af1f3b0022c1f7daa0c7d12ae72a21d.tar.gz
+
+[vars]
+filesize=219516
+sha512=8dd78f463811826729cb1498a7ccc4bff49393e88adb80260ad499623cf4ecdce5b405c1d91ca03f0b34bb7f5329124cb7a0541dbb880be2a634b057af93b9aa
+pkgver=1
+tardir=ncmpcpp-b7da5d1b9af1f3b0022c1f7daa0c7d12ae72a21d
+
+[deps]
+boost
+gnu-readline
+automake
+
+[deps.run]
+libmpdclient
+
+[build]
+
+[ -n "$CROSS_COMPILE" ] && \
+  xconfflags="--host=$($CC -dumpmachine) \
+  --with-sysroot=$butch_root_dir"
+
+# --------------------------------------------------------
+# NOTE TO USERS: clock and visualizer are enabled by default
+# on most distros, so they are to be expected. If you don't
+# wish to use them, these flags can be deleted:
+ADD_ONS="--enable-clock --enable-visualizer"
+# --------------------------------------------------------
+
+./autogen.sh
```

Comment:
why do you run autogen.sh ? if it's possible, we rahter avoid that, since automake is a very heavy dependency. if it is because there's no configure script in the git checkout, maybe the author has tagged releases which include proper release tarballs with included configure script (related info: https://github.com/sabotage-linux/sabotage/wiki/Why-github-downloads-suck )

Reply:
it does not ship with a configure script, and there isn't a "releases" page on his github.

The author has the configure script in his git-ignore. Looking back into commits of years ago you might be able to find a script but at that point its too old anyway.

Let me know what you want to do, and sorry if I misunderstood your comment

Reply:
that's how i do it: i raise awareness of the issue https://github.com/arybczak/ncmpcpp/issues/263

is there a specific reason you want to have latest master, over the latest tagged release from 2 weeks ago ? as i already mentioned, releases are usually preferable over bleeding egde master downloads.

Reply:
his releases page is here btw https://github.com/arybczak/ncmpcpp/releases
(click on code, then the releases button is visible)